flink/v1.20/flink/src/test/java/org/apache/iceberg/flink/source/enumerator/TestContinuousSplitPlannerImplStartStrategy.java

```diff
@@ -88,6 +88,23 @@ public void testTableScanThenIncrementalStrategy() throws IOException {
     assertThat(startSnapshot.snapshotId()).isEqualTo(snapshot3.snapshotId());
   }
 
+  @Test
+  public void testForLatestSnapshotStrategyExclusive() throws IOException {
```

Comment:
we should add another test method for non-empty table during start.

I realized this is also missing for the `testForLatestSnapshotStrategy` test below.